clcache.py

```diff
@@ -765,43 +776,110 @@ def expandCommandLine(cmdline):
     return ret
 
 
+class Argument(object):
+    def __init__(self, name):
+        self.name = name
+
+    def __len__(self):
+        return len(self.name)
+
+    def __str__(self):
+        return "/" + self.name
+
+    def __eq__(self, other):
+        return self.name == other.name
+
+    def __hash__(self):
+        return hash(self.name)
+
+
+# /NAMEparameter (no space, required parameter).
+class ArgumentT1(Argument):
+    def __init__(self, name):
+        super(ArgumentT1, self).__init__(name)
```

Comment:
Good idea, thanks.

I wanted to use those later on to implement `__eq__`  and `__hash__` on `Argument`, such that `ArgumentT1('W') != ArgumentT3('W')` but then did not do it.

I now use type() in `Argument` to get the behavior, such that I can use the `pass` as suggested.